MyHeartCounts/Heart Health Dashboard/HeartHealthDashboard.swift

```diff
@@ -207,30 +208,43 @@ extension HeartHealthDashboard {
 
 
 private struct HealthDashboardQuestionnaireView: View {
-    @Environment(\.locale)
-    private var locale
-    
     @Environment(MyHeartCountsStandard.self)
     private var standard
     
+    @Environment(StudyManager.self)
+    private var studyManager
+    
     @Environment(\.dismiss)
     private var dismiss
     
     let questionnaireName: String
+    @State private var questionnaire: Questionnaire?
     
     var body: some View {
-        if let questionnaire = Bundle.main.localizedQuestionnaire(withName: questionnaireName, for: locale) {
-            QuestionnaireView(questionnaire: questionnaire) { result in
-                switch result {
-                case .completed(let response):
-                    await standard.add(response: response)
-                case .cancelled, .failed:
-                    break
+        Group {
+            if let questionnaire {
+                QuestionnaireView(questionnaire: questionnaire) { result in
+                    switch result {
+                    case .completed(let response):
+                        await standard.add(response: response)
+                    case .cancelled, .failed:
+                        break
+                    }
+                    dismiss()
                 }
-                dismiss()
+            } else {
+                ContentUnavailableView("Unable to find Questionnaire", systemSymbol: .exclamationmarkTriangle) // ???
             }
-        } else {
-            ContentUnavailableView("Unable to find Questionnaire", systemSymbol: .exclamationmarkTriangle) // ???
         }
+        .task {
+            loadQuestionnaire()
+        }
+    }
+    
+    private func loadQuestionnaire() {
+        guard let studyBundle = studyManager.studyEnrollments.first?.studyBundle else {
+            return
+        }
+        questionnaire = studyBundle.questionnaire(named: questionnaireName, in: studyManager.preferredLocale)
```

Comment:
Nice idea to also package that one in the study bundle!

Reply:
we're also using the study bunde for the explainer texts we have below each of the metrics in the HHD; it's a useful mechanism